taxcalc/tests/test_policy.py

```diff
@@ -441,15 +391,7 @@ def test_policy_metadata():
     """
     clp = Policy()
     mdata = clp.metadata()
-    assert isinstance(mdata['STD']['value'], list)
-    assert np.allclose(mdata['STD']['value'],
-                       [6100, 12200, 6100, 8950, 12200])
-    assert isinstance(mdata['CDCC_ps']['value'], float)
-    assert mdata['CDCC_ps']['value'] == 15000
-    dump = False
-    if dump:
-        print(mdata)
-        assert 1 == 2
```

Comment:
`metadata()` is now just a wrapper for ParamTools' `Parameters.specification` method:

https://github.com/hdoupe/Tax-Calculator/blob/use-paramtools/taxcalc/parameters.py#L597